apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -90,18 +98,50 @@ export default class EnrollmentsPanel extends React.Component {
     })
       .done(() => {
         // reload
-        loadEnrollments();
+        this.handleEnrollmentRefresh();
         this.moveEnrollmentRequest = null;
       })
       .fail(() => {
         this.setState({
           error: 'Error: unable to move enrollments'
         });
-        loadEnrollments();
+        this.handleEnrollmentRefresh();
         this.moveEnrollmentRequest = null;
       });
   };
 
+  handleEditEnrollmentConfirmed = updatedName => {
+    this.handleEditEnrollment(updatedName, this.state.selectedEnrollments[0]);
+    this.setState({
+      enrollmentChangeDialogOpen: null,
+      selectedEnrollments: []
+    });
```

Comment:
When you confirm the edit, it looks like we're closing the dialog right away instead of waiting to see if the request succeeds or fails.  It might be preferable to leave the dialog up (and in a "pending" state) until the response comes back; if there's an error, it might be a better experience to show it inline in the dialog and leave it open, so the user can try again.

Reply:
It's the same behavior as the other dialog, and given low usage, I think I'll just leave it as-is. An error message appears in the heading of the enrollments list, which seems decent enough:

![image](https://user-images.githubusercontent.com/25372625/74961384-d9240a00-53c2-11ea-9284-36b51679d400.png)